spec/support/authenticated_session_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
I pushed a fix for the breakers integration spec that takes care of the problem by using the same code as what's provided here except exchanging expect for allow and removing the stipulations about the number of times it's called:

```ruby
allow_any_instance_of(ApplicationController).to receive(:authenticate_token).and_return(:true)
allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

It seems counterintuitive to me that this helper should enforce expectations that the method is called and that it should be called a specific number of times... given the name, I'd assume it would simply set up the stubbing so that the given user appears to be logged in without adding any expectations. So, if you wanted to switch the helper to work that way then you could change the breakers code back to use it.

I like your suggestion about making the breakers integration spec independent of the controller, but it's just so much easier to do it this way than with setting up a fake controller and adding routes for it and whatnot. If this becomes a frequent problem or it you have ideas about how to do that without a huge pain in the neck then I'm happy to spend some time on it.